runtime/lua/vim/treesitter/query.lua

```diff
@@ -255,15 +255,26 @@ local explicit_queries = setmetatable({}, {
 
 --- Sets the runtime query named {query_name} for {lang}
 ---
---- This allows users to override any runtime files and/or configuration
+--- This allows users to override or extend any runtime files and/or configuration
 --- set by plugins.
 ---
+--- For example, you could enable spellchecking of `C` identifiers with the
+--- following code (to try it, use `g==` or select the code then run `:'<,'>lua`):
```

Comment:
```suggestion
--- following code:
```

Reply:
Fair enough. I just added this as this is what `M.parse` does:
https://github.com/neovim/neovim/blob/bad3c9e5eccaf8d273bf52e68c82ae1c7dabf6a8/runtime/lua/vim/treesitter/query.lua#L346